lib/modules/apostrophe-assets/index.js

```diff
@@ -1288,7 +1305,10 @@ module.exports = {
         // this property won't be set
         middleware.push(self.lessMiddleware);
       }
-      middleware.push(self.apos.express.static(self.apos.rootDir + '/public'));
+      middleware.push(self.apos.express.static(
+        self.apos.rootDir + '/public',
+        _.defaults({}, self.options.minify === true ? self.options.serveStaticOptions : {}))
```

Comment:
I don't think we should be making assumptions about how others will use this option. It might be about caching — or something totally different.

Instead, developers should pass what they want to pass in each environment.

This can be done via the `data/local.js` mechanism, which merges the export of that file with the `app.js` apos configuration object, or via the use of environment variables etc. when writing configuration.